scripts/docker_cmake

```diff
@@ -31,8 +31,11 @@ ARGS=(
     -D Trilinos_ENABLE_ALL_OPTIONAL_PACKAGES=OFF
     -D Trilinos_ENABLE_TESTS=OFF
     -D Trilinos_ENABLE_EXAMPLES=OFF
+    -D Trilinos_ENABLE_OpenMP:BOOL=ON
     -D Trilinos_ENABLE_Tpetra=ON
         -D Tpetra_INST_INT_UNSIGNED_LONG=ON
+        -D Tpetra_INST_OPENMP:BOOL=ON
+    -D Kokkos_ENABLE_OpenMP:BOOL=ON
```

Comment:
Probably one of the `Trilinos_ENABLE_OpenMP` and `Kokkos_ENABLE_OpenMP` is not necessary.